package.json

```diff
@@ -1,13 +1,17 @@
 {
 	"scripts": {
 		"test": "xo",
-		"release": "npm run update-version && npm run upload",
-		"upload": "cd extension && webstore upload --auto-publish",
+		"release:amo": "cd extension && web-ext-submit",
+		"release:cws": "cd extension && webstore upload --auto-publish",
+		"release": "run-s update-version save-url release:*",
+		"save-url": "echo https://github.com/$TRAVIS_REPO_SLUG/tree/$TRAVIS_COMMIT > extension/SOURCE_URL",
```

Comment:
Where is this used?

Reply:
Nowhere, it's just a courtesy link to the source in each extension uploaded. Copy-pasted from Refined GitHun

Reply:
Hmm, I'm not sure how useful it is as it only shows the current commit. Wouldn't it be more useful with a link to comparison between last commit uploaded and the new one?

For example: https://github.com/sindresorhus/hide-files-on-github/compare/2.0.0...2.1.0

Reply:
Yeah the current commit is fine, it's not meant to be a changelog, it just points to the source of what's in the zip file — even though here there's no build (at the moment), so the source matches the zip contents. It can be removed I suppose

Reply:
hmmm, the generate link is: `https://github.com/sindresorhus/hide-files-on-github/tree/d181067`

It definitely points to the whole source _at the built commit_

Reply:
Ah, that's what you meant. Alright. I misunderstood.